2006-05-06  Joshua Kinard  <kumba@gentoo.org>

	* sysdeps/mips/fpu/feholdexcpt.c: Add libm_hidden_def.
	* sysdeps/mips/fpu/fesetround.c: Likewise.

--- ports/sysdeps/mips/fpu/feholdexcpt.c
+++ ports/sysdeps/mips/fpu/feholdexcpt.c
@@ -36,3 +36,4 @@ feholdexcept (fenv_t *envp)
 
   return 0;
 }
+libm_hidden_def (feholdexcept)
--- ports/sysdeps/mips/fpu/fesetround.c
+++ ports/sysdeps/mips/fpu/fesetround.c
@@ -41,3 +41,4 @@ fesetround (int round)
 
   return 0;
 }
+libm_hidden_def (fesetround)
